README.md

```diff
@@ -13,6 +13,14 @@ At first you need to install library
 npm install megaplanjs --save
 ```
 
+You can generate documentaion with commands:
+
+```
+npm install
+npm install jsdoc -g
```

Comment:
Do we really need to install it globally? Can we use devDependencies or `"docs-deps": "npm i jsdoc'` subtask for that?